src/client.rs

```diff
@@ -0,0 +1,46 @@
+use std::env;
+
+use anyhow::Result;
+
+use sea_orm::DatabaseConnection;
+use serenity::framework::standard::StandardFramework;
+use serenity::prelude::*;
+use songbird::SerenityInit;
+use tracing::{info, instrument};
+
+use crate::{command_handler::CommandHandler, store::guild_stores::GuildStores};
+
+#[instrument]
+pub async fn run(conn: DatabaseConnection) -> Result<()> {
+    let token = env::var("DISCORD_TOKEN").expect("DISCORD_TOKEN not set");
```

Comment:
Kanske lägga till "Missing DISCORD_TOKEN in env", för att förtydliga att den saknas i .env?

Reply:
Blir tydligare. Fixar!